src/common/export/download-svg.business.ts

```diff
@@ -24,8 +24,8 @@ export const downloadImage = (svg: JSX.Element, viewBoxSize: Size) => {
       // add scale factor to see the image clearly
       // Revisit this check #124
       // https://github.com/Lemoncode/mongo-modeler/issues/124
-      const scaleFactor = 2;
-      ctx.scale(scaleFactor, scaleFactor);
+      // const scaleFactor = 2;
+      // ctx.scale(scaleFactor, scaleFactor);
```

Comment:
I had to remove it becaouse otherwise when downloading the image was enlarged and  didnt work properly.

If its Ok this remove, we have to remove the issue and the comment how you can see in download-svg.business.ts